docs/s3.md

```diff
@@ -0,0 +1,75 @@
+# Performance Test Result storage in S3 bucket for SLV
+
+**NOTE: This document is only relevant to puppet employees**
+
+The SLV team has an s3 bucket for storing results from performance testing.
+
+`
+s3://slv-performance-results
+`
+
+## Convention
+
+S3 is [object storage](https://docs.aws.amazon.com/AmazonS3/latest/user-guide/using-folders.html),
```

Comment:
s/S3 is \[object storage\]\(...\),/S3 is an \[object storage\]\(...\) architecture,/